base/applications/cmdutils/at/lang/it-IT.rc

```diff
@@ -0,0 +1,63 @@
+/*
+ * PROJECT:     ReactOS AT Utility
+ * LICENSE:     See COPYING in the top level directory
+ * PURPOSE:     Italian resource file
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+LANGUAGE LANG_ITALIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "Il comando AT permette di stabilire altri comandi e programmi di eseguire\n\
+in un computer in un certo tempo. Lo servizio "Schedule" deve essere in esecuzione per\n\
+poter usare il comando AT.\n\n\
+AT [\\\\nomecomputer] [ [id] [/DELETE] | /DELETE [/YES]]\n\
+AT [\\\\nomecomputer] tempo [/INTERACTIVE]\n\
+    [ /EVERY:data[,...] | /NEXT:data[,...]] ""comando""\n\n\
+\\\\nomecomputer     Specifica un computer remoto. I comandi sono stabiliti in un \n\
+                   computer locale se questo parametro è omesso.\n\
+id                 È un numero d'identificazione assegnato in un comando\n\
+                   stabilito.\n\
+/DELETE            Cancella un comando stabilito. Se l'id viene omesso, tutti\n\
+                   i comandi stabiliti nel computer saranno cancellati.\n\
+/YES               Usato con l'opzione "DELETE", annullerà tutti i requisiti\n\
+                   dei comandi senza ulteriore conferma.\n\
+tempo              Specifica il tempo per l'esecuzione del comando.\n\
+/INTERACTIVE       Permette ai programmi di interagire con l'utente\n\
+                   registrato al momento dell'esecuzione pianificato.\n\
+/EVERY:data[,...]  Esegue il comando nel giorno o nei giorni specificati d'ogni\n\
+                   settimana o mese. Se la data è omessa, verrà considerata il\n\
+                   giorno attuale del mese.\n\
+/NEXT:data[,...]   Esegue il comando specificato nel prossimo giorno (per\n\
+                   esempio prossimo Martedì). Se la data è omessa, verrà\n\
+                   considerata il giorno attuale del mese.\n\
+""comando""          Il nome del comando o programma da eseguire.\n"
+
+    IDS_DELETE_ALL "Questa operazione cancellerà tutti i comandi stabiliti.\n"
+    IDS_NEW_JOB "È stato pianificato una nuova attività con l'ID = %lu\n"
+    IDS_JOBS_LIST "Stato ID   Giorno                     Tempo          Riga di comando\n"
+    IDS_NO_ENTRIES "Non ci sono comandi stabiliti nella lista.\n"
+
+    IDS_CONFIRM_QUESTION "Sei sicuro di continuare questa operazione? (S/N) [N]: "
```

Comment:
"di voler continuare"

Reply:
Oh my gosh... I forgot that one. :disappointed: